Makefile

```diff
@@ -174,9 +174,13 @@ endif
 #   make test-extended SUITE=core
 #   make test-extended SUITE=conformance FOCUS=pods
 SUITE ?= conformance
-FOCUS ?= .
+ifneq ($(strip $(FOCUS)),)
```

Comment:
Makes sure that if `FOCUS` is an empty string it is interpreted as null -- I remember in some code-path from Jenkins to `make` it was passed as such. Nobody wants to focus on only whitespace so I think this is reasonable.